runners/spark/src/main/java/org/apache/beam/runners/spark/aggregators/NamedAggregators.java

```diff
@@ -39,170 +41,185 @@
  * is launched. We can then add aggregators on the fly in Spark.
```

Comment:
@amitsela great write-up.

@bjchambers having the `NamedAggregators` in place, one challenge we're left with (addressed in this PR) is getting the values aggregated by `NamedAggregators` out to the various metrics collection backends (represented by Spark sinks).